src/components/auth/Login.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Icon, Input, Button } from 'antd';
+
+const LoginDetails = (props) => {
+  const { submit, form } = props;
+  return (
+    <Form onSubmit={event => submit(event, props.form)}>
```

Comment:
Since you have destructured your variables, could you consider accessing the directly?! For instance, instead of props.form, you could instead use form.

Reply:
Done